### ISSUE TITLE
Backup is skipped regardless of payload path being correct

### ISSUE THREAD

**Issue by favna** [Hamcha/lumaupdate#9]
This is an issue I have with 3 versions of the tool, the cia of Astronautlevel, the 3dsx release with custom cfg path again of Astronaut and the 1.0.0 release of the repo. When running the tool I get the output:

```
Original payload not found, skipping backup...
Downloadzing 7z file...

Update complete.
In case something goes wrong you can restore the old payload from /a9lh/ARN.bin.bak
Press START to reboot
```

As noted by the first line it is skipping making the backup, I assume this is related to handling the payload path for making the backup.

The cfg was in the following locations per situation:
- When using the cia it was on the root
- When using the 3dsx with custom cfg path support it was in `/aurei/`
- When using the 1.0.0 release it was on the root

My personal cfg has `payload path = /a9lh/ARN.bin` as content

**Comment by astronautlevel2** [comment]
Assuming @Hamcha doesn't get to it before I do I'll take a look at it in a few hours.

**Comment by astronautlevel2** [comment]
It appears that this issue was due to his configuration file having a leading slash. Removing the leading slash fixed this issue.

This can be closed now.

**Comment by favna** [comment]
Confirmed previous comment by @astronautlevel2. Closing issue.

**Comment by Hamcha** [comment]
The issue remains, arnupdate should only add the initial slash when omitted from the configuration file.

It's a bug, will be fixed in 1.1. (Should be easy to fix anyway)

**Comment by Hamcha** [comment]
Fixed in c07d231
